docs/administration/Configuration.md

```diff
@@ -133,6 +133,8 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 | Parameter                                     | Unit | Default                | Description                                                  |
 | --------------------------------------------- | ---- | ---------------------- | ------------------------------------------------------------ |
 | enable_strict_storage_medium_check            | -    | FALSE                  | Whether the FE strictly checks the storage medium of BEs when users create tables. If this parameter is set to `TRUE`, the FE checks the storage medium of BEs when users create tables and returns an error if the storage medium of the BE is different from the `storage_medium` parameter specified in the CREATE TABLE statement. For example, the storage medium specified in the CREATE TABLE statement is SSD but the actual storage medium of BEs is HDD. As a result, the table creation fails. If this parameter is `FALSE`, the FE does not check the storage medium of BEs when users create table. |
+| enable_auto_tablet_distribution               | -    | TRUE                   | Whether to automatically set the number of buckets. <ul><li> If this parameter is `TRUE`, you don't need to specify the number of buckets when a table is created or a partition is added. StarRocks automatically determines the number of buckets. For the strategy of automatically setting the number of buckets, see [Determine the number of buckets](../table_design/Data_distribution.md#determine-the-number-of-buckets).</li><li>If this parameter is set to `FALSE`, you need to manually specify the the number of buckets when a table is created or a partition is added. When a new partition is added and you don't specify the bucket count, the new partition inherits the number of buckets set when a table is created. However, you can also manually specify the number of buckets for the new partition.</li></ul>|
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/261737c8-0ddc-4eed-8757-1da6c3610cdd)
Weird logic. I can't see the logic behind this “however" sentence, because OF COURSE a user can manually specify the bucket count for a new partition regardless of whether this parameter is set to true or false. I think it's OK to delete this "however" sentence?